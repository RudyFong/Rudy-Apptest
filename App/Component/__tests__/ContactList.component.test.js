import React from 'react';
import renderer from 'react-test-renderer';
import ContactListComponent from '../ContactList.component';

jest.mock('../ContactList.component');
describe('ContactListComponent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <ContactListComponent navigateTo={jest.fn()} changeLanguage={jest.fn()}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
