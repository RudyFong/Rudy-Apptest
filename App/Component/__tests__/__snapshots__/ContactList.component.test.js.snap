// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactListComponent renders correctly 1`] = `null`;
